samples/boards/nrf/battery/src/battery.c

```diff
@@ -46,17 +46,25 @@ struct gpio_channel_config {
 struct divider_config {
 	const struct io_channel_config io_channel;
```

Comment:
These `const` modifiers here are actually redundant, as the whole instance of the structure is then created as `const`.